Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3075 @@
+{
```

Comment:
<p>In the whole NB, can you call <code>az.from_pymc3</code> directly in the <code>with</code> context please, just below <code>pm.sample</code> ?</p><p>Calling outside of the model context without passing <code>model</code> kwarg will be deprecated in a future version of ArviZ.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>